src/views/home.hbs

```diff
@@ -0,0 +1,74 @@
+
+    <!-- Showcase section -->
+    <div class="showcase">
+        <div class="container">
+            <div class="content">
+                <h1>Chat It Out </h1>
+                <h2>Gaza Psychological Service</h2>
+                <p>Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa.
+                    Cum sociis natoque penatibus et magnis
+                </p>
+                <button type="submit">Join Us</button>
+            </div>
+        </div>
+    </div>
+    <!-- Thereby conculdes the showcase section -->
+
+    <!-- Our services & contact us sections -->
+    <main>
+        <section>
+            <h1>Our Services</h1>
+            <div class="container2">
+                <div class="item">
```

Comment:
from this line 22 -47 you are using the same pattern where div>h1>p
with express handlebars you can use each loop and pass all titles and the paragraphs with object to the hbs file
```
{{#each obj}}
<div class="item"/>
    <h1>{{this.title}}</h1>
    <p>{{this.content}}</p>
</div>
{{/each}}
```

Reply:
WIll do in later projects

Reply:
@OnsNader **Then we will merge in later projects**